package.json

```diff
@@ -13,6 +13,8 @@
   "workspaces": [ "packages/*" ],
   "scripts": {
     "deploy:gh-pages": "lerna run build && ./script/deploy-gh-pages.js",
+    "lerna:remove:usage": "echo 'Usage: npm run lerna:remove -- module-to-remove \n cf. https://github.com/lerna/lerna/issues/833#issuecomment-396462481 \n cf. https://github.com/npm/npm/issues/9627#issuecomment-319449695 '",
+    "lerna:remove": "bash -c 'lerna exec -- yarn remove \"$0\" || lerna bootstrap'",
```

Comment:
Its on my Todo list to learn more about Lerna. 

I don't know what lerna:remove does yet.  Will find out.